apps/web/components/team/CreateTeamModal.tsx

```diff
@@ -25,6 +26,10 @@ export default function CreateTeamModal({ open, setOpen }: CreateTeamModalProps)
   const submitTeam = async (data) => {
     setLoading(true);
     const newTeam = await createTeam(data.name, (profile as any).id);
+
+    // Add demo product to the newly created team in the background
+    addDemoProduct(newTeam.id);
```

Comment:
same here, I think the `addDemoProduct` should happen inside the `createTeam` function.